common/src/main/java/com/radixdlt/monitoring/Metrics.java

```diff
@@ -234,6 +234,7 @@ public record LedgerSync(
       Counter remoteRequestsReceived,
       Gauge currentStateVersion,
       Gauge targetStateVersion,
+      Gauge targetProposerEpochSecond,
```

Comment:
I didn't understand what this was from just reading the name - I thought it was some diff between current and "target proposer" or something.

We call the thing being set the proposer timestamp. So perhaps we could call this metric `targetProposerTimestampEpochSecond`?

Reply:
Do you think we should add a comment that these two are untrusted?

Reply:
> We call the thing being set the proposer timestamp. So perhaps we could call this metric `targetProposerTimestampEpochSecond`?

I wonder if anything else than a `Timestamp` can have a unit of `EpochSecond` :p
nevertheless, lemme apply this rename suggestion 👍 

> Do you think we should add a comment that these two are untrusted?

For external readers? I've put it in https://github.com/radixdlt/babylon-node/pull/582/files#diff-28b98263fd5ec3641cf14cb342ed3dd3077c88c19fad7692a35b4ec8f8e680e3R92 (and now I see it might be nice to link to that doc from `Metrics.java` too)
For internal readers? I think all metrics in this file could use descriptions... TBH it would be best defined as Prometheus `HELP` string. We do have a nice support for defining it in Rust, but not in Java :( for now, let me put this single code comment here 👍

Reply:
> I wonder if anything else than a Timestamp can have a unit of EpochSecond :p
I read it as TargetProposerEpoch (as in Radix Epoch) Second (as in some time difference). So I got confused because "Epoch" typically means "Ledger Epoch" not "Unix Epoch" to me when working in the node codebase :D 

RE other stuff - code comments all sound good.